src/accelerate/utils/modeling.py

```diff
@@ -1465,6 +1469,13 @@ def infer_auto_device_map(
                 f"offload_buffers=True."
             )
 
+    for device, mem in device_minimum_assignment_memory.items():
```

Comment:
This has the potential to spam a lot of warnings. How about creating a single message and only warn once?

Reply:
Thanks for pointing that out! I have addressed the issue in my new commit.